config/default.yaml

```diff
@@ -143,6 +143,12 @@ bookends:
     teardown:
         - screwdriver-artifact-bookend
 
+notification:
```

Comment:
If this is supposed to be a stub setting, I think YAML expects you to initialize it as an empty object `{}`